OutOfSchool/OutOfSchool.WebApi/Controllers/V1/AddressController.cs

```diff
@@ -8,9 +8,11 @@
 using OutOfSchool.WebApi.Models;
 using OutOfSchool.WebApi.Services;
 
-namespace OutOfSchool.WebApi.Controllers
+namespace OutOfSchool.WebApi.Controllers.V1
 {
     [ApiController]
+    [ApiVersion("1.0")]
+    [Route("api/v{version:apiVersion}/[controller]/[action]")]
```

Comment:
Instead of changing each and every Controller couldn't routing be configured via [Conventional routing](https://docs.microsoft.com/en-us/aspnet/core/mvc/controllers/routing?view=aspnetcore-5.0#conventional-routing) ? (In `Startup.Configure`)